_posts/2022-11-10-release-1.15.3.md

```diff
@@ -0,0 +1,185 @@
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28960'>FLINK-28960</a>] -         Pulsar throws java.lang.NoClassDefFoundError: javax/xml/bind/annotation/XmlElement
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28975'>FLINK-28975</a>] -         withIdleness marks all streams from FLIP-27 sources as idle
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-28976'>FLINK-28976</a>] -         Changelog 1st materialization delayed unneccesarily
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29130'>FLINK-29130</a>] -         Correct the doc description of state.backend.local-recovery
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29138'>FLINK-29138</a>] -         Project pushdown not work for lookup source
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29205'>FLINK-29205</a>] -         FlinkKinesisConsumer not respecting Credential Provider configuration for EFO
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29207'>FLINK-29207</a>] -         Pulsar message eventTime may be incorrectly set to a negative number
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29253'>FLINK-29253</a>] -         DefaultJobmanagerRunnerRegistry#localCleanupAsync calls close instead of closeAsync
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29324'>FLINK-29324</a>] -         Calling Kinesis connector close method before subtask starts running results in NPE
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29325'>FLINK-29325</a>] -         Fix documentation bug on how to enable batch mode for streaming examples
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29381'>FLINK-29381</a>] -         Key_Shared subscription isn&#39;t works in the latest Pulsar connector
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29395'>FLINK-29395</a>] -         [Kinesis][EFO] Issue using EFO consumer at timestamp with empty shard
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29397'>FLINK-29397</a>] -         Race condition in StreamTask can lead to NPE if changelog is disabled
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29459'>FLINK-29459</a>] -         Sink v2 has bugs in supporting legacy v1 implementations with global committer
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29477'>FLINK-29477</a>] -         ClassCastException when collect primitive array to Python
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29479'>FLINK-29479</a>] -         Support whether using system PythonPath for PyFlink jobs
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29483'>FLINK-29483</a>] -         flink python udf arrow in thread model bug
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-29495'>FLINK-29495</a>] -         PulsarSinkE2ECase hang
```

Comment:
Remove this as per above: "(excluding improvements to the build infrastructure and build stability)"